src/lib/y2partitioner/format_mount/options.rb

```diff
@@ -23,47 +23,6 @@
 require "y2storage"
 
 module Y2Storage
-  # FIXME: Monkey patch of Y2Storage::PartitionId it should moved to
```

Comment:
zypper up before testing, like I did it. libstorage-ng also ignore versioning, so it is must have for any testing.